addons/purchase/wizard/purchase_line_invoice.py

```diff
@@ -30,6 +30,48 @@ class purchase_line_invoice(osv.osv_memory):
     _name = 'purchase.order.line_invoice'
     _description = 'Purchase Order Line Make Invoice'
 
+    def multiple_order_invoice_notes(self, orders):
```

Comment:
I would get rid of this function and replace by `" \n".join(order.notes for order in orders)`